sample/AspNetFullFrameworkSampleApp/MyModule1.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Elastic.Apm.DiagnosticSource;
+using System;
+using System.Web;
+using Elastic.Apm;
+
+namespace AspNetFullFrameworkSampleApp
+{
+	public class MyModule1 : IHttpModule
```

Comment:
@SergeyKleyman I think we renamed this... Are you sure we have all changes in this PR?

Reply:
Yeah, it's different from the other PR: https://github.com/elastic/apm-agent-dotnet/pull/112/files#diff-6fb69fd24db8b4a2903a7427ad18a0e6R6

Reply:
Yes, my bad :( I merged all the changes from FullFrameworkSample branch of main repo.

Reply:
👍